src/pages/auth/SellerRedirection.tsx

```diff
@@ -0,0 +1,50 @@
+import backgroundImage from '../../assets/register-login-background.svg';
+
+const SellerRedirection = () => {
+	return (
+		<div
```

Comment:
before rendering this component, first check if the **isAuthenticated** state is true and render the component. 

hint: you can select that state from store since we're registering it during login process

Reply:
Yeah  I solved it. I checked is Requires2FA is true because isAuthenticated is false at this step